renderdoc/driver/vulkan/vk_core.cpp

```diff
@@ -760,6 +760,16 @@ static const VkExtensionProperties supportedExtensions[] = {
     {
         VK_EXT_VALIDATION_FLAGS_EXTENSION_NAME, VK_EXT_VALIDATION_FLAGS_SPEC_VERSION,
     },
+#ifdef VK_IMG_filter_cubic
+    {
+        VK_IMG_FILTER_CUBIC_EXTENSION_NAME, VK_IMG_FILTER_CUBIC_SPEC_VERSION
+    },
+#endif
```

Comment:
The `VK_IMG_filter_cubic` extension isn't hard to add support for (just a few enums to pipe through to the right places), but it's not free. If you do want to whitelist support for it then you'll need to add handling to the right places, similar to PVRTC.